### DISCUSSION TITLE
32k模型RuntimeError: probability tensor contains either `inf`, `nan` or element < 0

### DISCUSSION THREAD

**Discussion by guodastanson** [zai-org/ChatGLM3#891] [Q&A]
跑的是官方demo，gradio或者streamlit都报这个，用综合模型里面启动也是报这个。do_sample我改了basic_demo下的cli_batch_request_demo.py里的，还是没效果。
环境是centos, python 3.10, cuda是12.1，GPU两张3090

**Comment by zRzRzRzRzRzRzR** [comment]
你正常对话能行吗，这个感觉是你出发了词表外的词
